File: App Escritorio/GestorJuego/SerializarJSON/Personaje.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SerializarJSON
{
    public class Personaje
    {
        public String nom { get; set; }
        public string frase { get; set; }

        public static string rutaImagen1 { get; set; }
        public static string rutaImagen2 { get; set; }
        public static string rutaImagen3 { get; set; }

        public Personaje()
        {

        }

        public Personaje(string nom, string frase)
        {
            this.nom = nom;
            this.frase = frase;
        }
    }
}

File: AppAndroid/App/app/src/main/java/com/example/joc/Personaje.java
package com.example.joc;

public class Personaje {

    //cada personaje tiene un nombre y una frase que se mostrara al final del juego en la pantalla Resultado
    private String nom;
    private String frase;

    //cada personaje tiene su ruta que sera static
    private static String rutaImagen1;
    private static String rutaImagen2;
    private static String rutaImagen3;

    //getters
    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getFrase() {
        return frase;
    }


}
